test/sample-test/run_sample_test.py

```diff
@@ -64,49 +62,42 @@ def check(self):
 
     ###### Create Job ######
     job_name = self._testname + '_sample'
-    ###### Test-specific arguments #######
-    if self._testname == 'tfx_cab_classification':
-      params = {
-        'output':
-          self._output,
-        'project':
-          'ml-pipeline-test',
-        'column-names':
-          'gs://ml-pipeline-dataset/sample-test/taxi-cab-classification/column-names.json',
-        'evaluation':
-          'gs://ml-pipeline-dataset/sample-test/taxi-cab-classification/eval5.csv',
-        'train':
-          'gs://ml-pipeline-dataset/sample-test/taxi-cab-classification/train20.csv',
-        'hidden-layer-size':
-          '5',
-        'steps':
-          '5'
-      }
-    elif self._testname == 'xgboost_training_cm':
-      params = {
-        'output': self._output,
-        'project': 'ml-pipeline-test',
-        'train-data': 'gs://ml-pipeline-dataset/sample-test/sfpd/train_20.csv',
-        'eval-data': 'gs://ml-pipeline-dataset/sample-test/sfpd/eval_5.csv',
-        'schema': 'gs://ml-pipeline-dataset/sample-test/sfpd/schema.json',
-        'rounds': '5',
-        'workers': '2'
-      }
+    ###### Figure out arguments from associated config files. #######
+    test_args = {}
+    try:
+      with open(DEFAULT_CONFIG, 'r') as f:
+        raw_args = yaml.safe_load(f)
+    except yaml.YAMLError as yamlerr:
+      raise RuntimeError('Illegal default config:{}'.format(yamlerr))
+    except OSError as ose:
+      raise FileExistsError('Default config not found:{}'.format(ose))
     else:
-      # Basic tests require no additional params.
-      params = {}
+      f.close()
```

Comment:
I think with statement will automatically close the f, won't it?

Reply:
Good catch. Fixed.